index.d.ts

```diff
@@ -50,3 +50,35 @@ export interface Statement {
 export interface StatementMap {
 	[ propertyId: string ]: Statement[];
 }
+
+export interface Term {
+	language: string;
+	value: string;
+}
+
+export interface TermList {
+	[language: string]: Term;
+}
+
+export interface AliasesList {
+	[language: string]: Term[];
+}
+
+export interface Fingerprintable {
+	labels: TermList;
+	descriptions: TermList;
+	aliases: AliasesList;
+}
+
+export interface Language {
```

Comment:
Where do the `Language`, `LanguageCollection`, and `LanguageTranslations` types come from? I don’t think we have entity JSON like that (the word “directionality” isn’t mentioned in the [JSON format documentation](https://doc.wikimedia.org/Wikibase/master/php/md_docs_topics_json.html)).

Reply:
It comes from the termbox datamodel https://gerrit.wikimedia.org/r/admin/repos/wikibase/termbox. I am working on this patch. https://phabricator.wikimedia.org/T255134 (Use WikibaseDataModelTypes instead of local Wikibase data model TS definitions)

Reply:
Well, as far as I can tell those interfaces are used for other data that doesn’t come from the Wikibase data model, so I don’t think they should be added in this repository / npm package.

Reply:
Language might only have these properties in termbox.

Reply:
oops, I accidentally re-requested a review.

Reply:
> Well, as far as I can tell those interfaces are used for other data that doesn’t come from the Wikibase data model, so I don’t think they should be added in this repository / npm package.

Ok. I will cancel this pull-request and make one only with the first commit.

Reply:
I have reverted the Language commit.